app/models/manageiq/providers/ibm_cloud/vpc/network_manager/cloud_subnet.rb

```diff
@@ -14,6 +15,47 @@ class ManageIQ::Providers::IbmCloud::VPC::NetworkManager::CloudSubnet < ::CloudS
     end
   end
 
+  def self.params_for_create(ems)
+    {
+      :fields => [
+        {
+          :component    => 'select',
+          :name         => 'cloud_network_id',
+          :id           => 'cloud_network_id',
+          :label        => _('Network'),
+          :isRequired   => true,
+          :includeEmpty => true,
+          :validate     => [{:type => 'required'}],
+          :options      => ems.cloud_networks.map do |cvt|
+            {
+              :label => cvt.name,
+              :value => cvt.id,
+            }
+          end
+        }
+      ]
+    }
+  end
+
+  def self.raw_create_cloud_subnet(ext_management_system, options)
```

Comment:
why is this a class method as opposed to an instance method like `raw_delete_cloud_subnet`?  If it were an instance method, the caller would not need to pass in `ext_management_system` and instead could use the `with_provider_connection` method

Reply:
@chessbyte since we're trying to create a cloud_subnet there is no instance record yet